### PR TITLE
Items for Overcoming Extreme Climatic Obstacles

### PR DESCRIPTION
**Pull request by ktksan**
This PR is a support PR for : https://github.com/Terasology/ManualLabor/pull/30
NOTE; the png textures are temporary and are to be treated as placeholders for final artwork.

### CODE REVIEW

**Comment by jdrueckert** [comment]
LGTM, the only thing I wondered: you created deer hide, but did you also adjust the deer to drop some? 🤔 Did I overlook that? If so, can you give me a quick pointer on where to look? 😅

**Comment by ktksan** [comment]
Nope you didn't miss it, I just forgot to add the file(deer.prefab) in here 😅, added now.

File: assets/prefabs/animals/deer.prefab [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "DropGrammar": {
     "blockDrops": [],
-    "itemDrops": ["2*WildAnimals:meat"]
+    "itemDrops": ["WildAnimals:deerHide"]
```
Comment (jdrueckert) [inline assets/prefabs/animals/deer.prefab]: Please don't overwrite the old item drop, I believe e.g. MR depends on deer dropping meat as a food source.
`DropGrammar` should allow you to
- either add multiple drops
- or (which I'd prefer) drop each of the two possible drops (meet, hide) with a certain probability

**Review by jdrueckert — CHANGES_REQUESTED** [review]

File: assets/prefabs/animals/deer.prefab (hunk 1 above)
Comment (ktksan) [reply assets/prefabs/animals/deer.prefab]: Yes I've added the two possible drops but both these drops will have their individual probability of drop which would be independent of each other, which means at any point of time, there can be either one of the two items, both items together, or even none at all.

File: assets/prefabs/items/deerHide.prefab [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "DisplayName": {
+        "name": "Deer Hide"
+    },
+    "CraftingStationIngredient": {
+        "type": "WildAnimals:deerHide"
+    },
+    "ItemHelp": {
+        "category": "WildAnimals",
+        "paragraphText": [
+             "Deer hide obtained after killing a deer."
```
Comment (jdrueckert) [inline assets/prefabs/items/deerHide.prefab]: `deerHide` has no information about when it's dropped. So something like "the skin of a deer" would be more suitable I'd say.

File: assets/prefabs/items/wool.prefab [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "DisplayName": {
+        "name": "Wool"
+    },
+    "CraftingStationIngredient": {
+        "type": "WildAnimals:Wool"
+    },
+    "ItemHelp": {
+        "category": "WildAnimals",
+        "paragraphText": [
+             "Wool obtained after killing a sheep."
```
Comment (jdrueckert) [inline assets/prefabs/items/wool.prefab]: see above, maybe rather "soft, thick hair that grows on the bodies of sheep"

**Review by jdrueckert — CHANGES_REQUESTED** [review]
Tip for item descriptions: if you have the item name, check that in an english-english dictionary (explains words using synonyms or descriptive phrases)

**Review by jdrueckert — APPROVED** [review]

**Pull request merged**